compiler/rustc_borrowck/src/diagnostics/region_errors.rs

```diff
@@ -681,39 +676,33 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
             ..
         } = errci;
 
-        let mut diag =
-            self.infcx.tcx.sess.struct_span_err(*span, "lifetime may not live long enough");
-
         let (_, mir_def_name) =
             self.infcx.tcx.article_and_description(self.mir_def_id().to_def_id());
 
+        let err = LifeTimeOutliveErr { span: *span };
```

Comment:
```suggestion
        let err = LifetimeOutliveErr { span: *span };
```
Still need to change `LifeTime` to `Lifetime` :)